src/transformers/models/albert/modeling_tf_albert.py

```diff
@@ -501,10 +448,10 @@ def __init__(self, config, add_pooling_layer=True, **kwargs):
         )
 
     def get_input_embeddings(self):
-        return self.embeddings
```

Comment:
does this still return the same type?

Reply:
I don't understand how it can be used above (line 420) in a `tf.matmul` if it's a layer and not a weight.